probe-rs/src/architecture/riscv/communication_interface.rs

```diff
@@ -689,8 +689,8 @@ impl RiscvCommunicationInterface {
         // assemble
         //  lb s1, 0(s0)
 
-        // Backup register s0
-        let s0 = self.abstract_cmd_register_read(&register::S0)?;
+        // Backup register s0 is the same address as the frame_pointer.
```

Comment:
Why does it matter that it is the frame pointer? We only back it up because we use it, I don't think it's relevant here that it is the frame pointer register. But now it sounds as if that's important.

Reply:
I actually added that comment, because I thought of replacing the `const` for S0, but decided against it, so I will delete this comment.